SPECS/llvm/llvm16.spec

```diff
@@ -29,7 +32,10 @@ The llvm-devel package contains libraries, header files and documentation
 for developing applications that use llvm.
 
 %prep
-%setup -q -n llvm-project-llvmorg-%{version}
+%autosetup -p1 -n llvm-project-llvmorg-%{version}
+
+echo "errors: FINDME"
+find -name SparseBufferRewriting.cpp
```

Comment:
Some debug code left over?
```suggestion
```

Reply:
Sure enough. Thanks for catching that, that's my bad.